### ISSUE TITLE
NullReferenceException when I click in the current position (blue dot)

### ISSUE THREAD

**Issue by LeeMaZong** [TorbenK/TK.CustomMap#309]
Hello Torben and thanks for your job.
I have an issue when I click on the blue dot in the current position on iOS

Here the Message log:
```
System.NullReferenceException: Object reference not set to an instance of an object
  at TK.CustomMap.iOSUnified.TKCustomMapRenderer.OnDidSelectAnnotationView (System.Object sender, MapKit.MKAnnotationViewEventArgs e) [0x00024] in <2db35de2eab74729b46d57fb7793cc08>:0 
  at MapKit.MKMapView+_MKMapViewDelegate.DidSelectAnnotationView (MapKit.MKMapView mapView, MapKit.MKAnnotationView view) [0x00014] in /Users/builder/data/lanes/5665/f70a1348/source/xamarin-macios/src/build/ios/native/MapKit/MKMapView.g.cs:1807 
  at (wrapper managed-to-native) UIKit.UIApplication:UIApplicationMain (int,string[],intptr,intptr)
  at UIKit.UIApplication.Main (System.String[] args, System.IntPtr principal, System.IntPtr delegate) [0x00005] in /Users/builder/data/lanes/5665/f70a1348/source/xamarin-macios/src/UIKit/UIApplication.cs:79 
  at UIKit.UIApplication.Main (System.String[] args, System.String principalClassName, System.String delegateClas
```

For information, I use the 2.00 stable version...
To wait for the solution, where can I overload the event in order to do nothing when I click on it?

Thanks for your help

**Comment by LeeMaZong** [comment]
For information, I use the 2.00 stable version...
To wait for the solution, where can I surcharge the event in order to do nothing when I click on it?

**Comment by TorbenK** [comment]
fixed in 2.0.1. Thanks for reporting
